### ISSUE TITLE
Did you mean to use FREE() instead of OS_FREE() ?

### ISSUE THREAD

**Issue by gchiu** [metaeducation/ren-c#527]
Under windows powershell and standard shell

out: copy ""
call/output "ls" out

and this crashes back to the windows powershell

**    Version:   2.102.0.3.40                                           **
**    Platform:  Windows win32-x64                                      **
**    Build:     2-Jun-2017/21:47:30                                    **
**    Commit:    2f6cbfac59261c5386c15e7941f6966e0b58c59a               **
**                                                                      **

**Comment by hostilefork** [comment]
In the beginning I tried to honor the OS_ALLOC() and OS_FREE() concept.  It was intended for cases when there was memory that the host would allocate that the core might free, or vice-versa.  I put in these checks because if one claims that the allocator could be anything, you shouldn't be free()ing something you OS_ALLOC()'d or OS_FREE()ing something you malloc'd().

Long-term I'm not sure if there are going to be any reasonable cases where the core and the host can't just exchange memory via GC'd values (like a BINARY!).  

In any case, what I did was change CALL to an extension which is relatively self-contained.  The concern about how to exchange memory between the core and the host is not an issue.  It can use whatever it likes.  So I just switched it to use malloc() and free() for its internal buffering.

https://github.com/hostilefork/rebol/commit/dda1d37ffdf7bcaf1b42e9d2869e8ccff74c4c2d

@zsx -- It further seems to me that the malloc() and free() shouldn't even generally be necessary, as it's just batching up data to be appended to the BINARY! or STRING! as they go.

I note also that there's `^M` in the output on Windows, which it seems we should avoid.  Perhaps allow them to stick around in the BINARY! case, as if you were asking for a binary you must have a reason and want to decode it yourself?
